src/pages/CartPage.js

```diff
@@ -5,32 +5,68 @@ export class CartPage {
     this.page = page; 
     this.cartListLocator = page.getByRole('list').nth(1);
 
-    this.espressoItem = this.cartListLocator.getByRole('listitem').filter({ hasText: 'Espresso'});
+    this.espressoItem = (
+      this
+      .cartListLocator
+      .getByRole('listitem').
+      filter({ hasText: 'Espresso'})
```

Comment:
```suggestion
      this.cartListLocator.getByRole('listitem').filter({ hasText: 'Espresso'})
```

no need to start with new line if it shoter than 80 symbols. or only last element is enough